### ISSUE TITLE
new Stride VS plugin is crashing visual studio

### ISSUE THREAD

**Issue by jeske** [stride3d/stride#720]
**Release Type**: Release
**Version**: (Launcher v4.0.6) (Stride VS Plugin v4.0.1)
**Platform(s)**: Windows
**Describe the bug**

Whenever I click on a Stride .sdsl file in visual studio 2019, it stalls and/or crashes Visual Studio. Presumably because of the new Stride plugin.

(updating to VS-Community 2019 v16.5.5 more reliably produces crashes instead of hangs)

**To Reproduce**
1. Install the stride plugin on Visual Studio 2019
2. Click on any .sdsl file
3. WITNESS: it stalls, and maybe crashes

**Expected behavior**
I expect it not to stall or crash. :)

I would be satisfied with a workaround explaining how to compile .sdsl files into .sdsl.cs files manually. Then I could uninstall the VS Plugin and use that process manually.

**Screenshots**

[stride plugin hanging visual studio - YouTube](https://www.youtube.com/watch?v=uFqVROkGrhI)

**Comment by Kryptos-FR** [comment]
For me it doesn't crash but I do get an error:
![image](https://user-images.githubusercontent.com/3006525/81769450-9b91d180-9518-11ea-8bb5-d371ca3180dd.png)

**Comment by xen2** [comment]
@Kryptos-FR You probably didn't compiled `Stride.VisualStudio.Commands` yet? (which is different from `Stride.VisualStudio.Package`)

@jeske Would something in Game Studio be good enough, or command-line preferred?

**Comment by Kryptos-FR** [comment]
@xen2 I didn't compile anything. I installed it from the launcher. I even removed any trace of locally compiled version, including from the nuget cache.

**Comment by xen2** [comment]
@Kryptos-FR Now you can do something like this:
`msbuild /restore /t:StrideAssetUpdateGeneratedFiles MaterialShader6.Game.csproj` to force `.sdfx`/`.sdsl` regeneration

**Comment by devjarmo** [comment]
I ran the upgrade, the launcher asked to install a new version of the launcher, I did that. Then I was asked to upgrade the VSX integration and engine. The VSX integration installer error'd out. Then all versions of the engine were then removed by the launcher..... I sighed. I went and then manually removed the VSX integration in VS then re-launched the launcher. Re-installed the VSX integration and the engine. Both completed successfully. However, now when I open VS and have Stride open, VS crashes. These upgrades have been painful. I'm not here to burden anyone but they are becoming a detractor from upgrading.

Any fix for VS crashing? Or do I just run with no VSX and not do an shaderwork?
